content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -156,24 +156,26 @@ already be running.
 You can view running containers (including static Pods) by running (on the node):
 ```shell
 # Run this command on the node where the kubelet is running
-docker ps
+crictl ps
 ```
 
 The output might be something like:
 
 ```
-CONTAINER ID IMAGE         COMMAND  CREATED        STATUS         PORTS     NAMES
-f6d05272b57e nginx:latest  "nginx"  8 minutes ago  Up 8 minutes             k8s_web.6f802af4_static-web-fk-node1_default_67e24ed9466ba55986d120c867395f3c_378e5f3c
+CONTAINER           IMAGE                                CREATED             STATE               NAME                        ATTEMPT             POD ID
```

Comment:
For these outputs, can you please compare to the rendered page at https://deploy-preview-31410--kubernetes-io-main-staging.netlify.app/docs/tasks/configure-pod-container/static-pod/#behavior-of-static-pods and reduce the column spacing? Thanks!

Reply:
There might be a bug of sorts here (possibly already filed, I haven't checked). The sample output isn't using the full width of the parent element, and that's not helpful for wide items like this one.

As a workaround, try appending `console` to line 164 @chris-short (I haven't tested this). The idea is to trigger the syntax highlighter.